cmd/fleetcli/main.go

```diff
@@ -3,6 +3,10 @@ package main
 
 import (
 	// Ensure GVKs are registered
```

Comment:
Please keep this comment together with the _blank imports_.

Reply:
should be fixed now